roomserver/storage/sqlite3/state_block_table.go

```diff
@@ -39,7 +39,8 @@ const stateDataSchema = `
 
 const insertStateDataSQL = "" +
 	"INSERT INTO roomserver_state_block (state_block_nid, event_type_nid, event_state_key_nid, event_nid)" +
-	" VALUES ($1, $2, $3, $4)"
+	" VALUES ($1, $2, $3, $4)" +
+	" ON CONFLICT DO NOTHING"
```

Comment:
Why did you add this?

Reply:
At this point I was trying to debug a problem in my database and should have removed this. Will fix.